Resources/Prototypes/_Carpmosia/Decals/bricktile_half_tech_maint_dark.yml

```
@@ -0,0 +1,83 @@
+﻿- type: decal
+  id: BrickTileTMaintDarkHalfCornerNe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_corner_ne
+
+- type: decal
+  id: BrickTileTMaintDarkHalfCornerSe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_corner_se
+
+- type: decal
+  id: BrickTileTMaintDarkHalfCornerNw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_corner_nw
+
+- type: decal
+  id: BrickTileTMaintDarkHalfCornerSw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_corner_sw
+
+- type: decal
+  id: BrickTileTMaintDarkHalfInnerNe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_inner_ne
+
+- type: decal
+  id: BrickTileTMaintDarkHalfInnerSe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_inner_se
+
+- type: decal
+  id: BrickTileTMaintDarkHalfInnerNw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_inner_nw
+
+- type: decal
+  id: BrickTileTMaintDarkHalfInnerSw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_inner_sw
+
+- type: decal
+  id: BrickTileTMaintDarkHalfN
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_n
+
+- type: decal
+  id: BrickTileTMaintDarkHalfS
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_s
+
+- type: decal
+  id: BrickTileTMaintDarkHalfE
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_e
+
+- type: decal
+  id: BrickTileTMaintDarkHalfW
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: tech_maint_dark_half_w    
```

```suggestion
    state: tech_maint_dark_half_w

```
You should keep newlines at the end of all files